readme.md

```diff
@@ -10,23 +10,21 @@ An [ESLint shareable config](https://eslint.org/docs/developer-guide/shareable-c
 ## Usage
 
 ```
-npm install --save-dev eslint-plugin-standard eslint-plugin-promise eslint-plugin-import eslint-plugin-node @typescript-eslint/parser @typescript-eslint/eslint-plugin eslint-config-standard-with-typescript 
+npm install --save-dev eslint-plugin-standard eslint-plugin-promise eslint-plugin-import eslint-plugin-node @typescript-eslint/eslint-plugin
```

Comment:
Both the config we extend from and the parser are `dependencies` of this package, so users should not install them.
#65.

Reply:
This is a nice improvement. Lowers the number of packages the user has to install.